.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,66 @@
+name: Feature Request
+description: Suggest an idea for NextAuth.js
+title: "[Feature Request]: "
+labels: enhancement
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for reaching out to us regarding the awesome feature that you believe should be included in the NextAuth.js library. Please provide the following information:
+  - type: input
+    id: summary
+    attributes:
+      label: Summary 💭
+      description: A clear and concise summary of the feature being proposed
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description 📓
+      description: Please provide a more in-depth description of the feature proposed.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Make sure you provide plenty of [links]() to external documentation and inline code examples like so:
+
+        ```js
+        function myAwesomeNextAuthFeature() {
+          return 💚
+        }
+        ```
+
+        Take time thinking about what you want to say and help us understand your proposal making sure that this description contains:
+
+        - **purpose of the feature**
+        - **potential problems**
+        - **potential alternatives**
+  - type: input
+    id: reproduction
+    attributes:
+      label: How to reproduce ☕️ 
+      description: If you have a CodeSandbox playground to help us visualize your idea better, please provide the link to it here.
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can use one of the templates set up on **CodeSandbox** to better illustrate your idea:
+
+        - [`next-auth-example`](https://codesandbox.io/s/next-auth-example-1kktb)
+        - [`next-auth-typescript-example`](https://codesandbox.io/s/next-auth-typescript-example-se32w)        
+  - type: markdown
+    attributes:
+      value: "## Contributing 🙌🏽"
+  - type: markdown
+    attributes: 
+      value: |
+        It takes a lot of work 🏋🏻‍♀️ maintaining a library like `next-auth`; any contribution is more than welcome 💚
+        In case you're willing to help implement this feature, please let us know here, and we'll reach you 😊 . Otherwise, you can have a look at the issues labelled with [`"good first issue"`](https://github.com/nextauthjs/next-auth/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and pick any of them.      
```

Comment:
It could be similar to the bug report template, we could ask a yes/no question if they are interested in implementing this feature, and also just refactor it the same way what we land on in the bug report template. I think the last sentence is unnecessary here as well.

Reply:
Makes sense.